feat: Add Flask 1.0.2 to tests and remove Python 2.6 and 3.3 as they reached EOL

galuszkak opened the pull request:
This PR adds to support Flask 1.0.2 and removing Python 2.6 and 3.3 as they reached EOL.

joshfriend commented:
Looks like `MethodView.methods` is now a `set` in Flask 1.x, which is causing test failures.

joshfriend commented on .travis.yml:
```diff
@@ -4,12 +4,11 @@ python:
   - 3.6
   - 3.5
   - 3.4
-  - 3.3
```
please also remove 2.6 and 3.3 from the `Makefile`, `setup.py` classifiers, `tox.ini`, `.python-version` and any docs where mentioned (`installation.rst` and `testing.rst` is where I also see it)

galuszkak commented:
@joshfriend I've incorporated Your feedback. Can You review it again?

Also because Python 3.7 was added I've added it to pipeline (it looks ugly but this is because it: https://github.com/travis-ci/travis-ci/issues/9815). Also added pypy and pypy3.

dhgutteridge commented:
Shouldn't the patch to Makefile also now include:

    +test-py37:
    +	PYTHON_MAJOR=3 PYTHON_MINOR=7 $(MAKE) test

galuszkak commented:
@dhgutteridge @joshfriend added.
